### ISSUE TITLE
Name of this repository

### ISSUE THREAD

**Issue by stephane-caron** [stack-of-tasks/pinocchio-minimal#6]
I mistakenly thought for a long time that "pinocchio-minimal" was a light version of Pinocchio itself, while it is actually a "minimum project" using Pinocchio. This leads me to believe the name can be improved :blush: 

What do you think of ``new_pinocchio_project`` (hyphenate to taste) as a name for this repository?

**Comment by nim65s** [comment]
I don't think this project is a "new pinocchio project", and this title might lead someone else to think that this is a new version of Pinocchio itself 😅 . 

I would rather call it an example or a template. But I'm calling everything an "example" these days…

**Comment by stephane-caron** [comment]
Ah right :smile: Well `pinocchio_template_project` (or just `pinocchio_template`?) sounds good too :+1:
